integration/kubernetes/untrusted_workloads/inyect_secret.yaml

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-secret
+data:
+  username: bXktYXBw
```

Comment:
- Is this value a username or a password? I'm confused. Maybe it's just a secret ;)
- Should this file be called `inject_secret.yaml`?